discover-rules/build.sbt

```diff
@@ -0,0 +1,4 @@
+name := "Discover Rules Job"
+version := "0.0.1"
+scalaVersion := "2.12.4"
```

Comment:
Looks like scala version should be 2.11 (2.11.11 works)

Reply:
2.12.4 works for me ... maybe I've got scala installed somewhere... I'll try 2.11.11.